Formula/wine.rb

```diff
@@ -134,6 +489,16 @@ def caveats
     s
   end
 
+  def post_install
+    # For fontconfig
+    ohai "Regenerating font cache, this may take a while"
+    system "#{libexec}/bin/fc-cache", "-frv"
+
+    # For net-snmp
+    (var/"db/net-snmp").mkpath
+    (var/"log").mkpath
+  end
+
   test do
     system "#{bin}/wine", "--version"
```

Comment:
Maybe tempting here to add a basic smoke test to ensure that `otool -L` shows `wine` linked correctly to the vendored tools in `libexec` given how large a change this is.